src/Catty/DataModel/Project/Project.m

```diff
@@ -660,4 +495,15 @@ + (NSString*)projectPathForProjectWithName:(NSString*)projectName projectID:(NSS
     return [NSString stringWithFormat:@"%@%@/", [Project basePath], [[self class] projectDirectoryNameForProjectName:[Util replaceBlockedCharactersForString:projectName] projectID:projectID]];
 }
 
+- (Scene *)copyScene {
```

Comment:
Please remove this method from the `Project` but instead implement `CBMutableCopying` in the `Scene`.